aixplain/factories/team_agent_factory/utils.py

```diff
@@ -40,6 +44,8 @@ def build_team_agent(payload: Dict, agents: List[Agent] = None, api_key: Text =
         llm_id=payload.get("llmId", GPT_4o_ID),
         use_mentalist=True if payload.get("plannerId", None) is not None else False,
         use_inspector=True if payload.get("inspectorId", None) is not None else False,
+        max_inspectors=payload.get("maxInspectors", 0),
```

Comment:
The default here is set to 0, while in `TeamAgentFactory` and `TeamAgent` is set to 1.